packages/driver/cypress/e2e/commands/cookies.cy.js

```diff
@@ -7,13 +7,13 @@ const isWebkit = Cypress.isBrowser('webkit')
 describe('src/cy/commands/cookies - no stub', () => {
   context('#getCookies', () => {
     it('returns cookies from the domain matching the AUT by default', () => {
-      cy.visit('http://localhost:3500/fixtures/generic.html')
+      cy.visit('http://www.barbaz.com:3500/fixtures/generic.html')
       cy.setCookie('foo', 'bar', { domain: 'www.foobar.com' })
       cy.setCookie('baz', 'qux', { domain: 'foobar.com' })
       cy.setCookie('foo', 'bar')
 
       cy.getCookies().then((cookies) => {
-        expect(cookies[0].domain).to.equal('localhost')
+        expect(cookies[0].domain).to.match(/\.?.barbaz\.com/)
```

Comment:
Yes, it does get returned because of [we match based on superdomain](https://github.com/cypress-io/cypress/blob/b595636d061d138cccc7d9a1e5f965145e995a6b/packages/server/lib/browsers/cdp_automation.ts#L65). I [updated the getCookies tests](https://github.com/cypress-io/cypress/pull/24471/commits/2277d99fa555f8fadba84269894a5c642252efbb) to check for that.

Reply:
OK so that also means cookies from `app.barbaz.com` would get returned, which seems consistent with itself. We can scope that work to #24526